actions/debootstrap_action.go

```diff
@@ -203,7 +223,11 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 		cmdline = append(cmdline, fmt.Sprintf("--variant=%s", d.Variant))
 	}
 
-	cmdline = append(cmdline, "--exclude=usr-is-merged")
+	// workaround for https://github.com/go-debos/debos/issues/361
+	if d.isLikelyOldSuite() {
+		cmdline = append(cmdline, "--exclude=usr-is-merged")
```

Comment:
can we print a warning here ?

e.g. `log.Printf("excluding usr-is-merged as package is not in suite\n")`

Reply:
All comments addressed. Compile-tested.